setup.py

```diff
@@ -99,6 +99,7 @@
     "accelerate>=0.26.0",
     "av==9.2.0",  # Latest version of PyAV (10.0.0) has issues with audio stream.
     "beautifulsoup4",
+    "blobfile==2.1.1",
```

Comment:
is there a need to pin a specific version for both blob and tiktoken?

Reply:
I'll check